typescript/ws-protocol/src/FoxgloveServer.ts

```diff
@@ -124,6 +124,15 @@ export default class FoxgloveServer {
     }
   }
 
+  /**
+   * Emit a time udpate to clients.
```

Comment:
```suggestion
   * Emit a time update to clients.
```

Reply:
seems to be a common mistake of mine :sweat_smile: